sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1266,6 +1266,38 @@ object functions {
   def rint(columnName: String): Column = rint(Column(columnName))
 
   /**
+   * Returns the value of the column `e` rounded to 0 decimal places.
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def round(e: Column): Column = round(e.expr, 0)
+
+  /**
+   * Returns the value of the given column `e` rounded to the value of `scale` decimal places.
+   *
+   * @group math_funcs
+   * @since 1.5.0
+   */
+  def round(e: Column, scale: Column): Column = Round(e.expr, scale.expr)
```

Comment:
As it expect the `Literal` type for 2nd argument of `Round`, we can remove this API.